src/AccordionItem/__snapshots__/accordion-item.spec.js.snap

```diff
@@ -54,25 +54,28 @@ exports[`AccordionItem renders correctly with accordion true 1`] = `
 </div>
 `;
 
-exports[`AccordionItem renders with different className 1`] = `
+exports[`AccordionItem renders correctly with other blocks inside 1`] = `
 <div
-  className="testCSSClass"
+  className="accordion__item"
 >
   <div
     ariaControls="accordion__body-mock-uuid-hash"
     expanded={false}
     id="accordion__title-mock-uuid-hash"
     onClick={[Function]}
-    role="tab"
+    role="button"
```

Comment:
It's just snapshots went a bit nuts. I double checked and everything looks fine to me. This one is not an accordion when the other one was.
I reckon if we add snapshots we should add them at the end so the existing ones are not reordered. Because the merge comparison looks messy after. Just a theory though.